### PR TITLE
wiki(meandeck): remove Ragavan content

### PR DESCRIPTION
**Pull request by angrybacon**

### CODE REVIEW

File: markdown/chapters/meandeck/alternative-piles.md [hunk 1]
```diff
@@ -117,26 +117,6 @@ As a result if they do cast an Endurance then you can pass and simply draw the
 second Oracle now and try to win with a Devotion of 4 with 3 cards remaining in
 your deck.
```
Comment (Doishy) [inline markdown/chapters/meandeck/alternative-piles.md:119]: We could leave this section in as a "play around Predict" example?

File: markdown/chapters/meandeck/interaction.md [hunk 2]
```diff
@@ -743,50 +742,6 @@ worst case scenario, could take your Oracle out of your hand leaving you unable
 to win. Just be mindful of when these cards could come up if building a PTT
 pile.
```
Comment (Doishy) [inline markdown/chapters/meandeck/interaction.md:744]: This section is fine but F my hard work wasted Q.Q

File: markdown/chapters/meandeck/matchups.md [hunk 3]
```diff
@@ -31,16 +31,6 @@ fast piles.
 ::accordion[Cards to keep in mind]{path=matchups/delver.ur.cards}
 ::accordion[Gameplay]{path=matchups/delver.ur.gameplay}
```
Comment (Doishy) [inline markdown/chapters/meandeck/matchups.md:33]: Can replace with Jeskai Days Undoing? Swap Ragavan line for Normal Jeskai shit?

File: markdown/chapters/meandeck/alternative-piles.md (hunk 1 above)
Comment (angrybacon) [reply markdown/chapters/meandeck/alternative-piles.md:119]: Not really (or would require reworking the whole line) Ragavan can only disrupt at sorcery speed but Predict is harder to play around with the given pile 🤔

File: markdown/chapters/meandeck/interaction.md (hunk 2 above)
Comment (angrybacon) [reply markdown/chapters/meandeck/interaction.md:744]: Ikr 😢 it's all on the history though forever remembered

File: markdown/chapters/meandeck/matchups.md (hunk 3 above)
Comment (angrybacon) [reply markdown/chapters/meandeck/matchups.md:33]: I'd rather do that in a separate MR (this one is strictly about purging Ragavan content and will be easier to search for/revert to)

**Pull request merged**